### PR TITLE
Topics now shows correct character count

### PR DESCRIPTION
**Pull request by Israellund**
## Link to Issue
Closes: #8372 

## Description of Changes
-User now sees correct character count when creating a new topic

## "How We Fixed It"

-added the label "Character count:" 
-created a function `getCharacterCount` to get the correct character count, as insert within Quill adds a trailing "/n", and would make the count +1 more than it actually was
## Test Plan
- Be an Admin and go into Topics
- create a new topic and give it a description
- -confirm that the character count actively changes and correctly corresponds with the amount of characters you have entered.

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/pages/CommunityManagement/Topics/CreateTopicsSection/CreateTopicSection.tsx [hunk 1]
```diff
@@ -112,9 +131,7 @@ export const CreateTopicSection = () => {
             setContentDelta={setDescriptionDelta}
           />
           <div className="description-char-count">
-            <CWText type="caption">
-              {descriptionDelta?.ops?.[0].insert.length / 250 || 0}
-            </CWText>
+            <CWText type="caption">Character count: {characterCount}</CWText>
```
[the remark this answers is not in the document]
Comment (Israellund) [reply packages/commonwealth/client/scripts/views/pages/CommunityManagement/Topics/CreateTopicsSection/CreateTopicSection.tsx:134]: Ah, good catch. I didn't know it had the "/250". That's why it was throwing the weird number but cause it was dividing by 250. Let me add the "/250" back in.

[the remark this answers is not in the document]
Comment (mzparacha) [reply packages/commonwealth/client/scripts/views/pages/CommunityManagement/Topics/CreateTopicsSection/CreateTopicSection.tsx]: thanks, i also noticed that user can enter > 250 chars and this bug is on master branch -- maybe we can tackle this in the pr or make (or a followup if you prefer)?

**Review by mzparacha — APPROVED** [review]

File: packages/commonwealth/client/scripts/views/pages/CommunityManagement/Topics/CreateTopicsSection/CreateTopicSection.tsx [hunk 2]
```diff
@@ -61,6 +62,24 @@ export const CreateTopicSection = () => {
...
+      }
+      return count;
+    }, 0);
+  };
+
+  useEffect(() => {
+    const count = getCharacterCount(descriptionDelta);
+    setCharacterCount(count);
+  }, [descriptionDelta]);
+
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/pages/CommunityManagement/Topics/CreateTopicsSection/CreateTopicSection.tsx:82]: it looks over complicated, especially that it worked some time ago. Could you try to revert this change and check if it works? 

https://github.com/hicommonwealth/commonwealth/commit/e6265d0a01495a4f70060c977d172ac245d41438#diff-0c09f2a69a0da3f96a9b614ad286aa003240b477153e110e3b34321d5d85fc5a

```
            <CWText type="caption">
              {descriptionDelta?.ops?.[0].insert?.length} / 250
            </CWText>
```

Comment (Israellund) [reply packages/commonwealth/client/scripts/views/pages/CommunityManagement/Topics/CreateTopicsSection/CreateTopicSection.tsx:82]: @masvelio I did try that. The problem with this is insert adds a "/n" character, and so the length count is always +1 of what the actual count was. And if you deleted all of the characters there would still be a count of 1.

Comment (masvelio) [reply packages/commonwealth/client/scripts/views/pages/CommunityManagement/Topics/CreateTopicsSection/CreateTopicSection.tsx:82]: thanks for checking this

**Review by masvelio — APPROVED** [review]
